Extra link in integration tests need to be removed

https://circleci.com/gh/mozilla/addons-frontend/19921

`firefox/channel/desktop` is the extra one , needs to be remove , breaking the tests 

```
    def test_browsers_footer_links(base_url, selenium, i, links):
        page = Home(selenium, base_url).open()
>       page.footer.browsers_links[i].click()
E       IndexError: list index out of range

```

--- a/tests/frontend/ui/test_home.py
+++ b/tests/frontend/ui/test_home.py
@@ -94,7 +94,6 @@ def test_add_ons_footer_links(base_url, selenium, i, links):
         'firefox/mobile',
         'mixedreality.mozilla.org',
         'firefox',
-        'firefox/channel/desktop',
     ])
 )
 @pytest.mark.nondestructive

